### DISCUSSION TITLE
WalletConnect v2 relay url seems blocked in some area

### DISCUSSION THREAD

**Discussion by hamberluo** [WalletConnect/walletconnect-monorepo#2776] [Developer Support]
relay url: wss://relay.walletconnect.org

**Comment by finessevanes** [comment]
> relay url: wss://relay.walletconnect.org

Are you experiencing this issue right now? Can you share what region you're in?

**Reply by hamberluo** [reply]
China

**Comment by tiennt0212** [comment]
I don't quite sure that we face the same problem, @hamberluo .

But I got a related issue: when I use my Metamask mobile application to scan the QR code, there's no connecting session was created. It also happens when I use WalletConnect on my smartphone.

I live in Vietnam, and I faced this issue about 9 hours ago. cc/ @finessevanes

**Reply by tiennt0212** [reply]
Additional information, I use Ethereum Provider in my migration

**Reply by finessevanes** [reply]
@tiennt0212 are you still experiencing this issue?

**Reply by tiennt0212** [reply]
Yes, I'm still experiencing this issue @finessevanes

**Reply by tiennt0212** [reply]
Here is my projectId: `b77a1626a05740d6a93b1707ac7b0761`

**Reply by tiennt0212** [reply]
I've created a separate ticket about that https://github.com/orgs/WalletConnect/discussions/2794 for better support. @finessevanes

**Comment by bobwith2bees** [comment]
I am in the US central region - so may not be related but I used to see relay connections last 300 seconds and reconnect reliably.  

Now I see a flapping behavior on alternate connections.   This may be limited to the Flutter package, but wanted to provide a data point if the team is trying to tune the load balancers.

```
I/flutter (14252): 18:37:25 🐛 WalletConnctSvc - Core.onRelayClientDisconnect - Marking relay connection down. Connection lasted 294 seconds.
I/flutter (14252): 18:37:30 🐛 WalletConnctSvc - Core.onRelayClientDisconnect - Marking relay connection down. Connection lasted 4 seconds.
I/flutter (14252): 18:42:26 🐛 WalletConnctSvc - Core.onRelayClientDisconnect - Marking relay connection down. Connection lasted 294 seconds.
I/flutter (14252): 18:42:31 🐛 WalletConnctSvc - Core.onRelayClientDisconnect - Marking relay connection down. Connection lasted 3 seconds.
I/flutter (14252): 18:47:27 🐛 WalletConnctSvc - Core.onRelayClientDisconnect - Marking relay connection down. Connection lasted 295 seconds.
I/flutter (14252): 18:47:32 🐛 WalletConnctSvc - Core.onRelayClientDisconnect - Marking relay connection down. Connection lasted 3 seconds.
I/flutter (14252): 18:52:28 🐛 WalletConnctSvc - Core.onRelayClientDisconnect - Marking relay connection down. Connection lasted 295 seconds.
I/flutter (14252): 18:52:33 🐛 WalletConnctSvc - Core.onRelayClientDisconnect - Marking relay connection down. Connection lasted 2 seconds.
I/flutter (14252): 18:57:30 🐛 WalletConnctSvc - Core.onRelayClientDisconnect - Marking relay connection down. Connection lasted 296 seconds.
I/flutter (14252): 18:57:34 🐛 WalletConnctSvc - Core.onRelayClientDisconnect - Marking relay connection down. Connection lasted 2 seconds.
I/flutter (14252): 19:02:32 🐛 WalletConnctSvc - Core.onRelayClientDisconnect - Marking relay connection down. Connection lasted 296 seconds.
I/flutter (14252): 19:02:35 🐛 WalletConnctSvc - Core.onRelayClientDisconnect - Marking relay connection down. Connection lasted 2 seconds.
I/flutter (14252): 19:07:33 🐛 WalletConnctSvc - Core.onRelayClientDisconnect - Marking relay connection down. Connection lasted 296 seconds.
I/flutter (14252): 19:07:36 🐛 WalletConnctSvc - Core.onRelayClientDisconnect - Marking relay connection down. Connection lasted 1 seconds.
I/flutter (14252): 19:12:34 🐛 WalletConnctSvc - Core.onRelayClientDisconnect - Marking relay connection down. Connection lasted 297 seconds.

```

**Reply by finessevanes** [reply]
Could you try updating to version 2.0.12 and let me know if the issue persists?

**Reply by bobwith2bees** [reply]
@finessevanes - Still seeing it on the Flutter side with .12.  Wrote up the issue here 

https://github.com/WalletConnect/WalletConnectFlutterV2/issues/116

I haven't confirmed on the wire if the relay actually sees the short connection.   So handling it in the Flutter issues for now.
